src/GlobalState.js

```diff
@@ -108,6 +109,8 @@ class Provider extends Component {
   setUserProfile = profile => {
     console.log('Current user', profile)
 
+    LogRocket.identify(profile)
```

Comment:
I think we should only send username (which is a public info) but not other user sensitive info.
We still don't know how LogRocket treat data and we don't want to pass too much data to them.

Reply:
Done